src/winter_onboarding_2021/fleet_management_service/handlers/cab.clj

```diff
@@ -0,0 +1,47 @@
+(ns winter-onboarding-2021.fleet-management-service.handlers.cab
+  (:require [ring.util.response :as response]
+            [clojure.spec.alpha :as s]
+            [winter-onboarding-2021.fleet-management-service.models.cab :as cab]
+            [winter-onboarding-2021.fleet-management-service.views.layout :as layout]
+            [winter-onboarding-2021.fleet-management-service.views.cab :as cab-views]))
+
+(s/def :cab/name string?)
+(s/def :cab/licence_plate string?)
+
+(s/def :cab/distance_travelled
```

Comment:
Why not use `number?`

Reply:
Or regex?

Reply:
The client is sending us `distance_travelled` in a string format. That's why we are checking for a string.